client/core/bond.go

```diff
@@ -477,22 +736,38 @@ func (c *Core) PostBond(form *PostBondForm) (*PostBondResult, error) {
 
 	var success bool
 
+	// When creating an account, the default is to maintain tier.
+	maintain := true
+	if form.MaintainTier != nil {
+		maintain = *form.MaintainTier
+	}
+
 	c.connMtx.RLock()
 	dc, acctExists := c.conns[host]
 	c.connMtx.RUnlock()
 	if acctExists {
 		if dc.acct.locked() { // require authDEX first to reconcile any existing bond statuses
 			return nil, newError(acctKeyErr, "acct locked %s (login first)", form.Addr)
 		}
+		if form.MaintainTier != nil || form.MaxBondedAmt != nil {
+			return nil, fmt.Errorf("maintain tier and max bonded amount may only be set when registering")
+		}
 	} else {
+		maxBondedAmt := 4 * form.Bond // default
```

Comment:
> `MaxBondedAmt` is a sort of safety feature?

Correct. Client posts bond to hit target tier, but it's server saying what tier you are, so if server keeps reporting low tier (either because penalization or malice), you'll keep posting bonds without limit.